unified-prototype/unified-plugin/gradle/libs.versions.toml

```diff
@@ -1,31 +1,40 @@
 [versions]
+# Kotlin Libs
 # Find latest version at: https://kotlinlang.org/docs/releases.html#release-details
 kotlin = "1.9.23"
 ksp = "1.9.23-1.0.20"
 
+# Android Libs
 # Find latest version at: https://developer.android.com/reference/tools/gradle-api
 android = "8.3.0"
+androidTools = "31.3.0"
 hilt = "2.50"
 room = "2.6.1"
-protobuf = "0.9.4"
 
+#Other Libs
 apache-commons = "3.3.1"
+dependency-guard = "0.4.3"
+truth = "1.4.2"
+protobuf = "0.9.4"
 
 [libraries]
-# Kotlin dependencies
+# Kotlin Libs
 kotlin-multiplatform = { module = "org.jetbrains.kotlin.multiplatform:org.jetbrains.kotlin.multiplatform.gradle.plugin", version.ref = "kotlin" }
 ksp-plugin = { module = "com.google.devtools.ksp:com.google.devtools.ksp.gradle.plugin", version.ref = "ksp" }
 kotlin-jvm = { module = "org.jetbrains.kotlin.jvm:org.jetbrains.kotlin.jvm.gradle.plugin", version.ref = "kotlin" }
 kotlin-serialization-plugin = { module = "org.jetbrains.kotlin:kotlin-serialization", version.ref = "kotlin" }
 
-# Android dependencies
+# Android Libs
 android-agp-application = { module = "com.android.application:com.android.application.gradle.plugin", version.ref = "android" }
 android-kotlin-android = { module = "org.jetbrains.kotlin.android:org.jetbrains.kotlin.android.gradle.plugin", version.ref = "kotlin" }
+android-tools-common = { module = "com.android.tools:common", version.ref = "androidTools" }
 hilt-android-plugin = { module = "com.google.dagger:hilt-android-gradle-plugin", version.ref = "hilt" }
 hilt-android = { module = "com.google.dagger:hilt-android", version.ref = "hilt"}
 hilt-compiler = { module = "com.google.dagger:hilt-android-compiler", version.ref = "hilt" }
-protobuf-plugin = { module = "com.google.protobuf:protobuf-gradle-plugin", version.ref = "protobuf" }
 room-plugin = { module = "androidx.room:room-gradle-plugin", version.ref = "room" }
 
-# Other libraries
+#Other Libs
 apache-commons-lang = { module = "org.apache.commons:commons-lang3", version.ref = "apache-commons" }
+dependency-guard-plugin = { module = "com.dropbox.dependency-guard:com.dropbox.dependency-guard.gradle.plugin", version.ref = "dependency-guard"}
```

Comment:
Seems odd to embed dropbox functionality into the android plugin

Reply:
Agreed.  I'm just mentally lumping this in with the other not-really-general-purpose-Android-plugin stuff that lives in our plugin for now, to support NiA.

Once we have a way to extend our SoftwareTypes with project-specific SoftwareTypes that could be defined in NiA, we should move this (along with the rest of the `NiASupport` functionality) back into the NiA project.